install/tests/cleanup.sh

```diff
@@ -35,4 +35,14 @@ for i in $(gsutil ls gs://nightly-tests/tf-state); do
     export TF_VAR_TEST_ID=$TF_VAR_TEST_ID
 
     make cleanup cloud=$cloud
+
+    CUSTOMERID=$(replicated customer ls --app "${REPLICATED_APP}" | grep "$TF_VAR_TEST_ID" | awk '{print $1}')
```

Comment:
Is there any way of making this a function that can be invoked? This is a repetition of what's in `.github/workflows/delete-kots-channel.yml`

Reply:
we ideally could, but I don't know if it is much of an optimization considering it means we will have to pass the API token, license name and in the very near future appname since I have an issue open to use gitpod-dev as published channel. Do you mind if I think about this better when I deal with that issue?

Reply:
Absolutely - let's get it merged